python/pyspark/sql/readwriter.py

```diff
@@ -373,6 +373,12 @@ def csv(self, path, schema=None, sep=None, encoding=None, quote=None, escape=Non
                        default value, ``false``.
         :param inferSchema: infers the input schema automatically from data. It requires one extra
                        pass over the data. If None is set, it uses the default value, ``false``.
+        :param enforceSchema: If it is set to ``true``, the specified or inferred schema will be
```

Comment:
can you add this option to streaming reader and writer?

Reply:
added